conf/shiro.ini.template

```diff
@@ -62,6 +62,12 @@ sessionManager = org.apache.shiro.web.session.mgt.DefaultWebSessionManager
 #cacheManager = org.apache.shiro.cache.MemoryConstrainedCacheManager
 #securityManager.cacheManager = $cacheManager
 
+cookie = org.apache.shiro.web.servlet.SimpleCookie
+cookie.name = JSESSIONID
+cookie.secure = true
```

Comment:
Also, I am not sure if you want to enable "secure" flag by default. As per OWASP, "Browsers which support the secure flag will not send a cookie with the secure flag set over an unencrypted HTTP request".  Refer (https://www.owasp.org/index.php/SecureFlag). This might cause an issue with installations where Zeppelin is running on plain HTTP.

However, httpOnly flag can be set by default and won't have any implications in both HTTP and HTTPS enabled Zeppelin.

In short comment this line.